content-src/selectors/selectors.js

```diff
@@ -68,10 +68,11 @@ module.exports.selectNewTabSites = createSelector(
     state => state.TopSites,
     state => state.FrecentHistory,
     state => state.History,
+    state => state.Highlights,
     selectSpotlight,
     state => state.Blocked
   ],
-  (TopSites, FrecentHistory, History, Spotlight, Blocked) => {
+  (TopSites, FrecentHistory, History, Highlights, Spotlight, Blocked) => {
```

Comment:
You need to remove `FrecentHistory` here and add `Highlights` to the `selectSpotlight` instead (line 36)

Alternatively, leave this selector as is and file a follow-up ticket